java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/utils/AccessorTestUtils.java

```diff
@@ -54,30 +67,16 @@ public int getCurrentRow() {
     }
   }
 
-  public interface AccessorSupplier<T extends ArrowFlightJdbcAccessor> {
```

Comment:
Please be careful with unrelated/unnecessary changes like this. This makes reviews harder